helpers/telemetryHelper.js

```diff
@@ -0,0 +1,140 @@
+const Telemetry = require('sb_telemetry_util')
+const telemetry = new Telemetry()
+const _ = require('lodash')
+const fs = require('fs')
+const path = require('path')
+appId = 'discussionMiddleware'
```

Comment:
Where are we using this constant? Why are we not using the same for Pdata?

Reply:
ok